src/ui/PrintableUri/PrintableUri.ts

```diff
@@ -17,7 +17,9 @@ export interface IPrintableUriOptions extends IResultLinkOptions {
 }
 
 /**
- * The PrintableUri component displays the URI, or path, to access a result.
+ * The PrintableUri component inherits from the ResultLink component and supports all of its options.
```

Comment:
I would also put back-ticks around PrintableUri (`PrintableUri`). And [`ResultLink`]{@link ResultLink} instead of just {@link ResultLink}. Otherwise, it's great :)